data_model/src/expression.rs

```diff
@@ -325,11 +348,13 @@ pub struct ContextValue {
 
 impl ContextValue {
     /// Number of underneath expressions.
+    #[inline]
     pub const fn len(&self) -> usize {
         1
     }
 
     /// Constructs `ContextValue`.
+    #[inline]
```

Comment:
Doesn't work because `String::from` is allocating. This function would never be inlined.